packages/p2p/src/pages/orders/orders.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useSafeState } from '@deriv/components';
 import { reaction } from 'mobx';
 import { observer } from 'mobx-react-lite';
-import OrderDetails from 'Components/order-details/order-details.jsx';
+import OrderDetails from 'Pages/orders/order-details';
 import { useStores } from 'Stores';
 import OrderTable from './order-table/order-table.jsx';
 import './orders.scss';
```

Comment:
also make sure to change the structure of the imports so general rule of thumb

React
Third party libraries
shared core components
P2P components
style files

```suggestion
import { reaction } from 'mobx';
import { observer } from 'mobx-react-lite';
import { useSafeState } from '@deriv/components';
import { useStores } from 'Stores';
import OrderDetails from 'Pages/orders/order-details';
import OrderTable from './order-table/order-table.jsx';
import './orders.scss';
```

Reply:
Its not part of the card, i will rearrange the order for this component in the other card later.